iaso/api/org_unit_change_requests/permissions.py

```diff
@@ -5,4 +5,13 @@
 
 class HasOrgUnitsChangeRequestPermission(permissions.BasePermission):
     def has_permission(self, request, view):
-        return request.user.is_authenticated and request.user.has_perm(iaso_permission.ORG_UNITS_CHANGE_REQUEST)
+        if request.user.is_superuser:
+            return True
+        return request.user.is_authenticated
```

Comment:
Nitpicking here but it can be simplified to:
```suggestion
        return request.user.is_authenticated
```

Reply:
Are you sure of your comment? Because `request.user.is_superuser` != `request.user.is_authenticated`

Reply:
If you are a superuser, you are authenticated, aren't you?

Reply:
Yes, but the inverse is not true.
Being authenticated doesn't imply having the [`is_superuser` flag](https://docs.djangoproject.com/en/3.2/ref/contrib/auth/#django.contrib.auth.models.User.is_superuser).

Reply:
I know, but if they are not a super user, it will return `is_authenticated`; it doesn't matter if the user has the `is_superuser` flag.

You can see this as :
- `is_superuser = a && b`
- `is_authenticated = a`

Your code does:
```python
if a && b:
    return True
return a
```
It doesn't matter what `b` (`is_superuser`) is since you'll always return the value of `a` (`is_authenticated`)

Reply:
I guess your suggestion is wrong then, what you mean is:

```python
return request.user.is_superuser or request.user.is_authenticated
```

Reply:
Ok, let's deconstruct this for the two cases:
A)
```python
if request.user.is_superuser:
     return True
return request.user.is_authenticated
```
B)
  ```python
return request.user.is_authenticated
```
```
|| Type of User          ||  A  ||   B  ||
| super user             | True  | True  |
| is authenticated       | True  | True  |
| is *not* authenticated | False | False |
```

As you can see, the condition about `is_superuser` is unnecessary.

Reply:
Ok got it, you're right.